bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -105,7 +111,10 @@ public void initialize() {
         updateStatus(ThingStatus.UNKNOWN);
         GrantServlet servlet = new GrantServlet(this, httpService);
         servlet.startListening();
-        this.grantServlet = servlet;
+        grantServlet = servlet;
+        scheduler.scheduleAtFixedRate(() -> {
+            requestCount = 0;
+        }, 0l, 1l, TimeUnit.HOURS);
```

Comment:
I have in mind something more clever that will update the channel at each new request with the number of requests executed during the last hour. My idea was not to reset the counter every hour.
This will require to store the requests timestamps, probably using a deque.
I will try to implement that and then push a proposal to your repo.

Reply:
I just pushed a proposal to you and my test is running.

Reply:
Merged it. Thanks for your idea, it's more powerful.